### DISCUSSION TITLE
Flux Deployment Not Ready

### DISCUSSION THREAD

**Discussion by mirfanfi** [fluxcd/flux2#4269] [General]
I have an issue where when I run the flux check command almost all deployments are not ready. 

`flux check
► checking prerequisites
✗ flux 2.1.0 <2.1.1 (new version is available, please upgrade) 
► checking controllers
✗ helm-controller: deployment not ready
► ghcr.io/fluxcd/helm-controller:v0.36.1
✗ image-automation-controller: deployment not ready
► ghcr.io/fluxcd/image-automation-controller:v0.36.1
✗ image-reflector-controller: deployment not ready
► ghcr.io/fluxcd/image-reflector-controller:v0.30.0
✗ kustomize-controller: deployment not ready
► ghcr.io/fluxcd/kustomize-controller:v1.1.0
✗ notification-controller: deployment not ready
► ghcr.io/fluxcd/notification-controller:v1.1.0`

does this make all the pods in the flux-system namespace disappear or not exist?

`k get pod -n flux-system
_No resources found in namespace flux-system_
`

**Comment by somtochiama** [comment]
Did you already bootstrap/install flux prior to running this command?

**Reply by mirfanfi** [reply]
Yeaa, already install bootstrap command, after that i try flux check
